backend/services/reaction.py

```
@@ -0,0 +1,77 @@
+        "replies": [build_reply_response(r, user) for r in replies]
+    }
+
+
+def build_reply_response(reply, user: User):
+    return {
+        "id": reply.id,
+        "content": reply.content,
+        "created_at": reply.created_at,
+        "user": serialize_user(reply.user),
+        "total_likes": len(reply.likes),
+        "user_liked": any(l.user_id == user.id for l in reply.likes)
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)